index.html

```diff
@@ -5,6 +5,22 @@
   <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
   <meta content="utf-8" http-equiv="encoding">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="Try PureScript is an online PureScript code editor for quickly experimenting with PureScript code snippets and ideas.">
+
+  <!--  Facebook  -->
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="http://try.purescript.org/">
+  <meta property="og:title" content="Try PureScript!">
+  <meta property="og:image" content="http://try.purescript.org/img/channel_icon-800.png">
```

Comment:
One last thing - could you please add an attribution notice in a comment here? Something like

```html
<!-- 
The PureScript Logo by Gareth Hughes is licensed under the Creative Commons Attribution 4.0 license (https://creativecommons.org/licenses/by/4.0/).
-->
```

Thanks!

Reply:
Or maybe you could use `twitter:image:alt`.

Reply:
Sure, we should add this info!